app/models/container_image_registry.rb

```diff
@@ -4,4 +4,10 @@ class ContainerImageRegistry < ActiveRecord::Base
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   has_many :container_images, :dependent => :destroy # What about deleted registry but containers are still running
   has_many :containers, :through => :container_images
+
+  virtual_column :host_port, :type => :string
+
+  def host_port
+    port.nil? ? host : host + ":" + port
```

Comment:
port as a String will probably be a problem in the future as it has in the past.  It's expected that port is a number.

Maybe change this conditional to be positive and use interpolation?

```ruby
port.present? ? "#{host}:#{port}" : host
```

Also, is this `host_port` method used in building URIs? Because it's not IPv6 compatible.